src/lib.rs

```diff
@@ -138,6 +138,59 @@ impl Overlay {
     }
 }
 
+/// For when you can't give away ownership of the window.
+pub struct BorrowedOverlay {
+    active: Option<bool>,
+    pub active_opacity: u8,
+    pub inactive_opacity: u8,
+}
+
+impl BorrowedOverlay {
+    pub fn new(active_opacity: u8, inactive_opacity: u8) -> Self {
+        Self {
+            active: None,
+            active_opacity,
+            inactive_opacity,
+        }
+    }
+
+    pub fn toggle(&mut self, window: &Window) {
+        if !self.is_active() {
+            self.activate(window)
+        } else {
+            self.deactivate(window)
+        }
+    }
+
+    /// Make the overlay clickable.
+    pub fn activate(&mut self, window: &Window) {
+        if !self.is_active() {
+            self.active.replace(true);
+            platform_impl::make_window_overlay_clickable(&window);
+            platform_impl::set_window_overlay_opacity(&window, self.active_opacity);
+        }
+    }
+
+    /// Make the overlay transparent to inputs.
+    pub fn deactivate(&mut self, window: &Window) {
+        if self.is_active() {
+            self.active.replace(false);
+            platform_impl::make_window_overlay(window);
+            platform_impl::set_window_overlay_opacity(window, self.inactive_opacity);
+        }
+    }
+
+    /// Returns true if the overlay can be interacted with through mouse clicks.
+    ///
+    /// # Remarks
+    ///
+    /// This will return `true` even when [`BorrowedOverlay::activate`] has never been called, as the
+    /// `Window` that will eventually be used with the [`BorrowedOverlay`] will be clickable by the mouse.
+    pub fn is_active(&self) -> bool {
+        self.active.unwrap_or(true)
```

Comment:
When I call activate() right after new(), the window is not set active opacity.

Reply:
My intuition was that overlays should be "inactive" by default, but I guess there's room for disagreement here.
I'll just add the `active` flag to the constructor, then.

Reply:
Thank you for taking my opinion.

Reply:
No problem. You're currently benefiting more from this library than I am.

Reply:
I feel like I have no more issues. But sorry I don't have a button to mark these resolved. I don't have permission maybe?  Could you do this instead of me?

![image](https://user-images.githubusercontent.com/29710855/103484853-38e99f00-4e35-11eb-8010-501a5fa02008.png)